### ISSUE TITLE
markdown cell broken by simple latex: \[

### ISSUE THREAD

**Issue by RafalSkolasinski** [jupyter/notebook#3393]
Markdown cell gets broken (not possible to write, view) if following content is included:
````
```latex
\[
````

**Comment by ax3l** [comment]
following [this post on twitter](https://twitter.com/S4_117/status/970695256697163779), `$` seems to be problematic as well

**Comment by smartsammler** [comment]
@RafalSkolasinski In some issues I read that pandoc is used for converting markdown to LaTeX and pandoc needs to be given the extension
`+tex_math_single_backslash` to the markdown-extension to interpret the `\[`- and `\(`-syntax and not just the old `$$`- and `$`-syntax.
So unless Jupyter converts with 
```
pandoc -f markdown+tex_math_single_backslash -t latex $filename
```
only the `$`-syntax will be supported I guess.

I also would like to use this syntax in markdown-cells

**Comment by takluyver** [comment]
Pandoc is not involved here; that's used by nbconvert when we convert a whole notebook to a tex file. When you're editing a notebook in the browser, CodeMirror is the JS text editor component, and marked is responsible for rendering markdown, with Mathjax handling a subset of latex syntax.

Opening the browser's JS console shows me an error `state.localMode.indent is not a function`, which looks like https://github.com/jupyterlab/jupyterlab/issues/2923 and https://github.com/codemirror/CodeMirror/issues/4947 . That has already been fixed in CodeMirror, and PR #3372 updated the version we use.

I've just rebuilt the JS and tested, and it seems to be working for me now, so this should be fixed when we release notebook 5.5. Does anyone else want to install from source ([instructions](http://jupyter-notebook.readthedocs.io/en/stable/contributing.html#setting-up-a-development-environment)) to double check that?
